File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT1101Q01FwkDoctorInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroOUT1101Q01FwkDoctorInfo {
	private String sabun;
    private String doctorName;
	public NuroOUT1101Q01FwkDoctorInfo(String sabun, String doctorName) {
		super();
		this.sabun = sabun;
		this.doctorName = doctorName;
	}
	public String getSabun() {
		return sabun;
	}
	public void setSabun(String sabun) {
		this.sabun = sabun;
	}
	public String getDoctorName() {
		return doctorName;
	}
	public void setDoctorName(String doctorName) {
		this.doctorName = doctorName;
	}
	
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT1101Q01FwkDoctorInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroOUT1101Q01FwkDoctorInfo
    {
        public NuroOUT1101Q01FwkDoctorInfo() {}
    
        private string _sabun = "";
        public string Sabun
        {
            get { return _sabun; }
            set { _sabun = value; }
        }
        private string _doctorName = "";
        public string DoctorName
        {
            get { return _doctorName; }
            set { _doctorName = value; }
        }
    }
}